src/foam/nanos/fs/fileDropZone/FileDropZone.js

```diff
@@ -207,6 +211,7 @@ foam.CLASS({
               .end();
             }))
         .end()
+        .start().show(this.showHelp$)
```

Comment:
Or, if this is never going to be dynamically updated, use callIf() instead to avoid adding invisible DOM. Is there a button somewhere to toggle showing help?